client/src/Components/Common/Footer/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import WhatsAppIcon from '@mui/icons-material/WhatsApp';
+import Typography from '@mui/material/Typography';
+import './style.css';
+
+function Footer() {
+  return (
+    <div className="container">
```

Comment:
```suggestion
    <div className="footer-container">
```
having too many container classes, will lead to intervention other styles together